x-pack/metricbeat/module/meraki/network_health/networks.go

```diff
@@ -0,0 +1,124 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package network_health
+
+import (
+	"strconv"
+
+	sdk "github.com/meraki/dashboard-api-go/v3/sdk"
+
+	"github.com/elastic/beats/v7/metricbeat/mb"
+	"github.com/elastic/beats/v7/x-pack/metricbeat/module/meraki"
+	"github.com/elastic/elastic-agent-libs/mapstr"
+)
+
+// ID is the unique identifier for all networks
+type ID string
+
+// Network contains attributes, statuses and metrics for Meraki networks
+type Network struct {
+	id       ID
+	name     string
+	vpnPeers *[]sdk.ResponseItemApplianceGetOrganizationApplianceVpnStatsMerakiVpnpeers
+}
+
+func reportNetworkMetrics(reporter mb.ReporterV2, organizationID string, networks map[ID]*Network) {
+	metrics := []mapstr.M{}
+	for _, network := range networks {
+		if network == nil || network.vpnPeers == nil {
+			continue
+		}
+
+		metric := mapstr.M{
+			"network.id":   network.id,
+			"network.name": network.name,
+		}
+
+		peers := make([]mapstr.M, len(*network.vpnPeers))
+		for i, peer := range *network.vpnPeers {
+			peers[i] = vpnPeerToMapStr(&peer)
+		}
+		metric["network.vpn_peers"] = peers
+
+		metrics = append(metrics, metric)
+	}
+
+	meraki.ReportMetricsForOrganization(reporter, organizationID, metrics)
+}
+
+func vpnPeerToMapStr(peer *sdk.ResponseItemApplianceGetOrganizationApplianceVpnStatsMerakiVpnpeers) mapstr.M {
+	res := mapstr.M{
+		"network_id":   peer.NetworkID,
+		"network_name": peer.NetworkName,
+	}
+
+	if peer.UsageSummary != nil {
+		if recv, err := strconv.Atoi(peer.UsageSummary.ReceivedInKilobytes); err == nil {
+		    res["usage_summary.received.kb"] = recv
```

Comment:
```suggestion
		    res["usage.received.bytes"] = recv*1024
```
To adhere to naming convention